src/components/home/skills.tsx

```diff
@@ -19,23 +19,65 @@ const StyledHeading = styled.h2`
   font-weight: 500;
   @media all and (max-width: ${breakpoints.md}px) {
     grid-column: 1 / span 12;
-    font-size: ${getRemSize(dimensions.headingSizes.medium.mobile)};
+    font-size: ${getRemSize(dimensions.headingSizes.large.mobile)};
   }
 `;
 
 const StyledSkillsWrapper = styled.div`
-  grid-column: 3 / span 8;
+  grid-column: 1 / span 12;
   display: flex;
   flex-direction: column;
   align-items: center;
 `;
 
+const SkillsMobile = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-between;
+
+  img {
+    height: auto;
+    margin: 13px;
+  }
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    img {
+      width: 180px;
+    }
+  }
+
+  @media (max-width: 700px) {
+    img {
+      width: 136px;
+    }
+  }
+
+  @media (max-width: 555px) {
+    img {
+      width: 105px;
+    }
+  }
+
+  @media (max-width: 440px) {
+    img {
+      width: 77px;
+    }
+  }
+`;
+
+const StyledText = styled.p`
+  font-size: ${getRemSize(dimensions.headingSizes.cta.mobile)};
+  text-align: center;
```

Comment:
Needs a little bit of spacing between these elements

```suggestion
  text-align: center;
  margin-top: 37px;
```